examples/complex/src/routes.js

```diff
@@ -12,5 +12,18 @@ export default () => (
         <IndexRoute component={ Main } value={true}/>
         <Route path="about/" component={ About } data={2} />
         <Route path="simple/" component={ Simple } data={3} />
+        <Route
+            path="async"
+            getComponent={ (nextState, callback) => {
+                require.ensure([], () => {
```

Comment:
I think we should use the `require` function that we get back from `require.ensure`.

```js
require.ensure([], (require) => {
 const Async = require('./components/async').default;
```